pkgs/by-name/gv/gvmd/package.nix

```
@@ -0,0 +1,136 @@
+{
+  cmake,
+  doxygen,
+  fetchFromGitHub,
+  git,
+  glib,
+  gnutls,
+  gpgme,
+  graphviz,
+  gvm-libs,
+  icu,
+  lib,
+  libbsd,
+  libical,
+  libidn2,
+  libpkgconf,
+  libselinux,
+  libsepol,
+  libtasn1,
+  libxslt,
+  openvas-scanner,
+  ospd-openvas,
+  p11-kit,
+  perl538Packages,
+  pcre2,
+  pkg-config,
+  postgresql,
+  stdenv,
+  util-linux,
+  xmltoman,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gvmd";
+  version = "23.4.0";
+
+  src = fetchFromGitHub {
+    owner = "greenbone";
+    repo = "gvmd";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-Bxq6o98viMMwt2eoiZgPDTYqD/z6jRWYLgH/ZJvkpV4=";
+  };
+
+  prePatch = ''
+    substituteInPlace src/sql_pg.c \
+      --replace-fail "#include <postgresql/libpq-fe.h>" "#include <${postgresql}/include/libpq-fe.h>"
+    substituteInPlace doc/CMakeLists.txt \
+      --replace-fail "share/doc/gvm/html/" "\''${GMP_DIR}/doc/gvm/html/" \
+      --replace-fail "share/man/man8/" "\''${GVMD_MAN_DIR}/man/man8/"
+  '';
+
+  configurePhase = ''
+    runHook preConfigure
+
+    cmake -DSYSCONFDIR=$out/etc \
+      -DBINDIR=$out/bin \
+      -DSBINDIR=$out/sbin \
+      -DLIBDIR=$out/lib \
+      -DLOCALSTATEDIR=$out/var \
+      -DINCLUDEDIR=$out/include \
+      -DGMP_DIR=$out/share \
+      -DGVMD_MAN_DIR=$out/share \
+      -DDATADIR=$out/share \
```

```suggestion
      -DDATADIR=$out/share \
```
Data dir in the nix store? Are you sure? Is it stateful?

---

without it result to https://termbin.com/bdhv